### ISSUE TITLE
Milk properties should be dynamic

### ISSUE THREAD

**Issue by JoeWaddell** [RuminantFarmSystems/RuFaS#962]
Milk protein `mPrt` and lactose `lactose_milk` are hardcoded in `__init__` in `cow.py`, and never again modified. Likewise crude protein (?) `CP_milk`. 

## Possible Solution
The minimum improvement would be to set these values to 0.0 or None when not lactating, and then to the currently hard-coded values when lactation begins.

The more in-depth solution would be to modify those values dynamically, either due to expected changes during a lactation period, or due to an animal's health, contingent on what another SME thinks.

**Comment by KFosterReed** [comment]
Closing this as resolved by animal refactor

**Comment by tomhuhh** [comment]
Milk fat% and milk true protein % do not currently appear to change dynamically (e.g. respond to changes during a lactation period), but I could be mistaken.

**Comment by JoeWaddell** [comment]
> Milk fat% and milk true protein % do not currently appear to change dynamically (e.g. respond to changes during a lactation period), but I could be mistaken.

Correct, they're static, and based off user inputs. That's something that changed since opening this issue (that values were hardcoded in python files), so perhaps I should update the description and reopen this? Pending Kristan's input on closing this for now.

**Comment by KFosterReed** [comment]
Yes, they are not dynamic but are based on a user input. I think the intention behind this issue was that they appeared to be dynamic and “updated” with the animal’s lactation. This has changed and they are now clearly static and based on user input. A future issue could be opened with a proposal for how to implement a truly dynamic milk fat and protein value.
